packages/components/src/lib/controllers/app-controller.js

```diff
@@ -85,17 +82,7 @@ export default class AppController {
     this.resourceOptionsDefaults = LanguageOptionDefaults
     this.uiOptionsDefaults = UIOptionDefaults
     this.siteOptionsDefaults = SiteOptions
-    /*
-    All following options will be created during an init phase.
-    This will allow creators of an app controller to provide their own options defaults
-    inside a `create()` builder function.
-     */
-    this.featureOptions = null
-    this.resourceOptions = null
-    this.uiOptions = null
-    this.siteOptions = null // Will be set during an `init` phase
 
-    this.irregularBaseFontSize = !AppController.hasRegularBaseFontSize()
```

Comment:
do you remember what this was for? It seems it isn't used but I can't remember why we had it.

Reply:
I don't remember exactly, but probably before we've started to use CSS vars for font size we were setting font size in inline styles. This code was checking what the value was and if it was any different from the default value.

Reply:
ok